samples/snippets/csharp/language-reference/builtin-types/EnumType.cs

```diff
@@ -71,7 +71,7 @@ public static void Main()
                 var b = (Season)1;
                 Console.WriteLine(b);  // output: Summer
 
-                var c = (Season)4;
+                var c = (int)Season.Winter;
```

Comment:
That would make `c` equal 3, not 4 as intended. Intention here is to show that one can convert *any* integer to the enum type.

Reply:
Sorry about that, meant to change result to 3.

Reply:
Then it becomes the example of converting enum to int. That's already covered by the first example. I don't think any changes are required here.